src/evaluate/evaluator/token_classification.py

```diff
@@ -119,7 +121,8 @@ def prepare_data(self, data: Union[str, Dataset], input_column: str, label_colum
             references = data[label_column]
 
         metric_inputs = {"references": references}
-        pipeline_inputs = [join_by.join(element) for element in data[input_column]]
+        data = data.map(lambda x: {input_column: join_by.join(x[input_column])})
```

Comment:
Out of curiosity: is this faster than a list comprehension?

Reply:
I don't think so, but I will not load the data into memory.